agent-control/src/instrumentation/config/logs/format.rs

```diff
@@ -15,6 +15,22 @@ impl Default for TimestampFormat {
     }
 }
 
+/// Represents the supported logging formatters
+#[derive(Debug, Deserialize, PartialEq, Clone)]
+#[serde(rename_all = "snake_case")]
+pub enum Formatter {
+    /// Human-readable single-line logs
+    Default,
+    /// Newline delimited JSON logs
+    Json,
+}
+
+impl Default for Formatter {
+    fn default() -> Self {
+        Self::Default
+    }
+}
```

Comment:
If we decided to default to `Json` we would need a better name for the `Default` formatter 😅

Any suggestion is more than welcome!

Reply:
We can call the default `Pretty` or `HumanText` or something like this?

Also remember that you can set up the `Default` impl for an enum with:

```suggestion
/// Represents the supported logging formatters
#[derive(Default, Debug, Deserialize, PartialEq, Clone)]
#[serde(rename_all = "snake_case")]
pub enum Formatter {
    /// Human-readable single-line logs
    #[default]
    Default,
    /// Newline delimited JSON logs
    Json,
}
```

Reply:
Thanks! I totally forgot about the `#[default]` for enums. Probably `pretty` or `human_text` are better names 🤔

Reply:
I've updated the name (and also made `Json` the default). Let me know what you think 🙂